.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,100 @@
+    validations:
+      required: true
+  - type: textarea
+    id: repro-steps
+    attributes:
+      label: Steps to Reproduce
+      description: Describe all the steps we need to take to show the behavior that you have observed.
+      placeholder: | 
+        1. Create a File > New .NET MAUI App
+        2. Add a `Button` like so: `<Button Text="this is a bug" />`
+        3. Click the added button and observe the bug 🐞
+    validations:
+      required: true
+  - type: dropdown
+    id: version-with-bug
+    attributes:
+      label: Version with bug
+      description: In what version do you see this issue?
+      options:
+        - Preview 7
+        - Preview 8
+        - Preview 9
+        - Preview 10
+        - RC 1
+        - Unknown/Other (please specify)
+    validations:
+      required: true
+  - type: dropdown
+    id: version-that-worked
+    attributes:
+      label: Last version that worked well
+      description: Is there a version on which this _did_ work? If yes, which one? If no or unknown, please select `Unknown/Other`.
+      options:
+        - Preview 7
+        - Preview 8
+        - Preview 9
+        - Preview 10
+        - RC 1
+        - Unknown/Other
+    validations:
+      required: true
+  - type: dropdown
+    id: platforms-affected
+    attributes:
+      label: Affected platforms
+      description: Select all or any platform that you see this issue on. This helps us determine if it's something platform-specific or in the core. If you were only able to test on 1 platform, please check the last option to inform us about that.
+      multiple: true
+      options:
+        - iOS
+        - Android
+        - Windows
+        - macOS
+        - Other (Tizen, Linux, etc. not supported by Microsoft directly)
+        - I was *not* able test on other platforms
+    validations:
+      required: true
+  - type: input
+    id: platform-versions
+    attributes:
+      label: Affected platform versions
+      description: Please provide the version numbers of the platforms you see this issue on.
+      placeholder: E.g. iOS 15, Android 11 and up, Windows SDK 10.0.17134.0, etc.
+    validations:
+      required: true
+      email: true
+  - type: textarea
+    id: workaround
+    attributes:
+      label: Did you find any workaround?
+      description: Did you find any workaround for this issue? This can unblock other people while waiting for this issue to be resolved or even give us a hint on how to fix this.
+  - type: textarea
+    id: logs
+    attributes:
+      label: Relevant log output
+      description: Please copy and paste any relevant log output. This will be automatically formatted into code, so no need for backticks.
+      render: shell
+  - type: checkboxes
+    id: terms
+    attributes:
+      label: Code of Conduct
+      options:
+        - label: I agree to follow this project's [Code of Conduct](https://github.com/dotnet/maui/blob/main/.github/CODE_OF_CONDUCT.md)
+          required: true
```

Comment:
Does this need to be a required option, or could we just link to it and not require a checkbox? I feel like this would be an annoyance for the 99.999% of people who have no problem with this anyway.

Reply:
I think that's fair. We can get rid of the checkbox and rephrase so it says "By submitting this issue, I am agreeing to follow this project's Code of Conduct"

Reply:
Yes I think that would be perfect. That statement already holds true by participating in the repo, so this is just a reminder, as opposed to an extraneous step.

Reply:
That's a great suggestion! Technically they already agree whenever they participate and sign the CLA I think. I just love to have some reminders from time to time to people that the CoC is there and that there is actual people on the other side of the GitHub issue :)

But definitely think checking the box each time can become very annoying. This is probably a good middle-ground. Thank you!

Reply:
Fixed 
![image](https://user-images.githubusercontent.com/939291/141266368-97987957-138e-4e08-bb24-9f59dfd98400.png)